code/game/machinery/doors/firedoor.dm

```diff
@@ -1,6 +1,12 @@
 #define CONSTRUCTION_PANEL_OPEN 1 //Maintenance panel is open, still functioning
 #define CONSTRUCTION_NO_CIRCUIT 2 //Circuit board removed, can safely weld apart
 #define DEFAULT_STEP_TIME 20 /// default time for each step
+#define MINIMUM_TEMPERATURE_TO_BURN_ARMS 500 ///everything above this temperature will start burning unprotected arms
+#define TOOLLESS_OPEN_DURATION_SOLO 20 SECONDS ///opening a firelock without a tool takes this long if only one person is doing it
+#define RIGHT_ARM_DEAL_BURN if(right_arm && right_arm.status == BODYPART_ORGANIC && !right_arm.is_pseudopart) \
+				right_arm.receive_damage(0, 5)
+#define LEFT_ARM_DEAL_BURN if(left_arm && left_arm.status == BODYPART_ORGANIC && !left_arm.is_pseudopart) \
+				left_arm.receive_damage(0, 5)
```

Comment:
No, not this. The NUMBERS should be defines. This doesn't help maintainabiltiy at all.

Reply:
ah oops